src/views/user/LogEntryCardsView/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from 'react';
+import LogEntryList from './LogEntryList';
+import { getlogEntry } from '~/redux/slices/log-entry';
+import { PATH_APP } from '~/routes/paths';
+import { makeStyles } from '@material-ui/core/styles';
+import { useDispatch, useSelector } from 'react-redux';
+import Page from '~/components/Page';
+import { Container, Avatar, Box } from '@material-ui/core';
+import HeaderDashboard from '~/components/HeaderDashboard';
+import DateDisplay from '~/components/Date/date';
+
+// ----------------------------------------------------------------------
+
+const useStyles = makeStyles(theme => ({
+  root: {}
+}));
+
+// ----------------------------------------------------------------------
+
+function LogEntryCardsView() {
+  const classes = useStyles();
+  const dispatch = useDispatch();
+  const logEntry = useSelector(state => state.logEntry);
+
+  useEffect(() => {
+    dispatch(getlogEntry());
+  }, [dispatch]);
+
+  return (
+    <Page title="Management | User Cards" className={classes.root}>
+      <Container>
+        <HeaderDashboard
+          heading="Log Entries"
+          links={[
+            { name: 'Dashboard', href: PATH_APP.root },
+            { name: 'Management', href: PATH_APP.management.root },
+            { name: 'User', href: PATH_APP.management.user.root },
+            { name: 'Log entries' }
+          ]}
+        />
+        <Box display="flex">
+          <Avatar sx={{ marginRight: 2 }}>R</Avatar>
```

Comment:
According to the US Acceptance criteria, I should be able to see the profile's picture of the child we're visiting. 

```
Background: Carolina logged in the application
  Given: I have clicked on a specific child's card.
  And:  There are some pumper/meal/nap log entries for today
 --->>>>> And: The child's picture is visible at the top of the list <<<--------

```